src/hardware/serialport/softmodem.cpp

```diff
@@ -150,6 +150,16 @@ CSerialModem::CSerialModem(const uint8_t port_idx, CommandLine *cmd)
 		        GetPortNumber(), telnet_mode ? "enabled" : "disabled");
 	}
 
+	// Get the connect speed to report
+	if (getUintFromString("baudrate:", val, cmd)) {
+		val = val > 0 && val <= 57600 ? val : 57600;
+		LOG_MSG("SERIAL: Port %" PRIu8 " will report baud rate %d",
+			GetPortNumber(), val);
+	} else {
+		val = 57600;
```

Comment:
`57600` is a hardcoded magic number; suggest lifting it to a self-documenting constant (something like):

```c++
constexpr auto max_baudrate = 57600u;
if (getUintFromString("baudrate:", val, cmd)) {
	val = val > 0 && val <= max_baudrate ? val : max_baudrate;
	LOG_MSG("SERIAL: Port %" PRIu8 " will report baud rate %d",
		GetPortNumber(), val);
} else {
	val = max_baudrate;
}
```

Reply:
https://www.techradar.com/news/internet/getting-connected-a-history-of-modems-657479